Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js

```diff
@@ -1,62 +1,50 @@
-/* eslint-disable */
 function details() {
   return {
-    id: "Tdarr_Plugin_x7ac_Remove_Closed_Captions",
-    Stage: "Pre-processing",
-    Name: "Remove closed captions",
-    Type: "Video",
-    Operation: "Remux",
+    id: 'Tdarr_Plugin_x7ac_Remove_Closed_Captions',
+    Stage: 'Pre-processing',
+    Name: 'Remove burned closed captions',
+    Type: 'Video',
+    Operation: 'Remux',
     Description:
-      "[Contains built-in filter] If detected, closed captions (XDS,608,708) will be removed.",
-    Version: "1.00",
+      '[Contains built-in filter] If detected, closed captions (XDS,608,708) will be removed from streams.',
+    Version: '1.01',
     Link:
-      "https://github.com/HaveAGitGat/Tdarr_Plugins/blob/master/Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js",
-    Tags: "pre-processing,ffmpeg,subtitle only",
+      'https://github.com/HaveAGitGat/Tdarr_Plugins/blob/master/Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js',
+    Tags: 'pre-processing,ffmpeg,subtitle only',
   };
 }
 
 function plugin(file) {
-  //Must return this object
-
-  var response = {
+  const response = {
     processFile: false,
-    preset: "",
-    container: ".mp4",
+    preset: ',-map 0 -codec copy -bsf:v "filter_units=remove_types=6"',
+    container: '.${file.container}',
```

Comment:
This won't work, it has to be backticks. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thank you!